### DISCUSSION TITLE
No Tuya devices showing up in HA

### DISCUSSION THREAD

**Discussion by FWanderydz** [tuya/tuya-home-assistant#835] [Q&A]
Hi,

I have recently installed HA and the done they Tuya integration without issues. Authentication works, I see my devices in Tuya IOT platform and it gets updated correctly, but the Tuya integration in HA is empty so I see no devices there. 
Anyone that has any ideas what the issue can be?
![HA-Tuya](https://user-images.githubusercontent.com/45487055/185063414-78f05496-ca57-4cb8-934b-43e5e681e594.gif)

**Comment by jbduche** [comment]
_Hi,
Same problem ...
Maybe my Tuya product https://expo.tuya.com/product/1043042 is not yet supported by the integration ?
I have the message "error while get mqtt config" in the HA server logs ...
Wait for a solution ...
Thanks_

**Reply by ryanwaldron** [reply]
Getting the exact same result. All I have are simple lights, they all show up when I add the mobile app in the dev portal....but the devices never show up in HA.

**Reply by wangkai911220** [reply]
have the same problem. Added to HA successfully before and one day it just disappeared.
